decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb

```diff
@@ -47,21 +47,32 @@ module.exports = {
         3: "#E1E5EF",
         4: "#242424",
         5: "#F6F8FA",
-        6: "#D3D5D9"
+        6: "#D3D5D9",
+        7: "#D4D4D4",
+        8: "#737373",
+        9: "#F5F5F5"
       },
       background: {
         DEFAULT: "#F3F4F7",
         2: "#FAFBFC",
         3: "#EFEFEF",
         4: "#E4EEFF99", // 60% opacity
-        5: "#E9E9E9"
+        5: "#E9E9E9",
```

Comment:
This should be removed and its usages replaced with `neutral-200` (#E9E9E9 and #E5E5E5 are almost the same color as far as I see) 

<img width="321" height="144" alt="Image" src="https://github.com/user-attachments/assets/5648af4a-e726-4083-9a40-9dbb6d66ce45" />

<img width="341" height="151" alt="Image" src="https://github.com/user-attachments/assets/5400b883-534a-47a6-9b60-30d2084210af" />

https://www.color-hex.com/color/E9E9E9
https://www.color-hex.com/color/E5E5E5

Reply:
Added a reminder here: https://github.com/decidim/decidim/issues/15299#issuecomment-3380196515